convert2rhel/data/6/x86_64/configs/suse_es-6-x86_64.cfg

```diff
@@ -0,0 +1,10 @@
+[system_info]
+
```

Comment:
```suggestion

# SUSE ES packages are not signed
gpg_fingerprints =
```
This would cause an error due to the `gpg_fingerprints` key not being found:
https://github.com/oamg/convert2rhel/blob/29fd6a298e6d3a0c02db967dcb26202042ef9de6/convert2rhel/systeminfo.py#L130